driver-core/src/main/com/mongodb/client/model/BulkWriteOptions.java

```diff
@@ -31,6 +32,7 @@ public final class BulkWriteOptions {
     private Boolean bypassDocumentValidation;
     private BsonValue comment;
     private Bson variables;
+    private BsonDocument sort;
```

Comment:
Judging by the [spec](https://github.com/qingyang-hu/specifications/blob/master/source/crud/crud.md), BulkWriteOptions does not seem to require a sort option.

Reply:
I overlooked that part, will remove it!